src/main/java/com/bouncestorage/swiftproxy/v1/AccountResource.java

```diff
@@ -151,6 +153,8 @@ public BulkDeleteResult bulkDelete(@NotNull @PathParam("account") String account
         }
 
         BulkDeleteResult result = new BulkDeleteResult();
+        Map<String, List<String>> removeBlobsMap = new HashMap<>();
+        List<String> deleteContainers = new LinkedList<>();
```

Comment:
`ArrayList` is almost always better even for small collections due to various Java object overhead.